pkg/store/sandbox/sandbox.go

```diff
@@ -31,7 +31,9 @@ type Sandbox struct {
 	Metadata
 	// Containerd sandbox container
 	Container containerd.Container
-	// TODO(random-liu): Add cni network namespace client.
+	// CNI network namespace client
+	//TODO(Abhi): Should we handle multi network scenario
```

Comment:
It was a note to myself to see if there is a case where pod can be attached to multiple networks. Then how does sandbox hold the network namespace.